en/releases/daily_builds.md

```diff
@@ -10,9 +10,12 @@ These can be downloaded from the links below (install as described in [Download
 * [Windows](https://s3-us-west-2.amazonaws.com/qgroundcontrol/builds/master/QGroundControl-installer.exe)
 * [OS X](https://s3-us-west-2.amazonaws.com/qgroundcontrol/builds/master/QGroundControl.dmg)
 * [Linux](https://s3-us-west-2.amazonaws.com/qgroundcontrol/builds/master/QGroundControl.AppImage)
-* [Android](https://qgroundcontrol.s3-us-west-2.amazonaws.com/builds/master/QGroundControl.apk) (APK download) <!-- (Google Play) -->
-  > **Note** *QGroundControl* is temporarily unavailable from the Google Play Store. 
-    We are working on a fix for this, but it may take some time.
-    In the meantime you can use the APK supplied above.
+* Android is available from the Google Play store. 
```

Comment:
Is it still at the same link? If so, suggest you replace this with the old text:

```
[Android](https://play.google.com/store/apps/details?id=org.mavlink.qgroundcontrolbeta) (Google Play)
```

I would also put the note below in the bullet

Reply:
Yeah, I'll fix that.

Reply:
I couldn't get the Note in the bullet and get formatting given that and the other bullets that associate with the Note. That said, probably because I don't know enough about Markdown to make it work.

Reply:
Yes, this is challenging. Essentially this is indentation, plus sometimes you have to put in additional `>` marks to connect the lines. I've done this here: https://github.com/mavlink/qgc-user-guide/commit/b3b16177a2730f74193da993affa10a1ff7315ba

Reply:
Though note that something is going on with the gitbook server, and this has not built yet.